azure/vm.sh

```diff
@@ -614,7 +641,7 @@ fn azure_vm_backup_delete(backup_resgroup) {
 #
 # This function returns an empty string on success or a
 # non empty error message if it fails.
-fn azure_vm_backup_recover(instance, backup_resgroup) {
+fn azure_vm_backup_recover(instance, storagesku, backup_resgroup) {
 
 	fn log(msg) {
```

Comment:
Yeah, but there is code from the time that print was not available and then code from the time that print was bugged =P, so some parts still do not use it.